Makefile

```diff
@@ -21,9 +21,12 @@ docs-clean:
 
 lint:
 	flake8 --ignore=E731 .
-	pylint -j $(CPU_COUNT) --reports=n --disable=I --ignore-imports=y \
+	pylint -j $(CPU_COUNT) --reports=n -E \
```

Comment:
Used `-E` to raise only Errors and fatal errors, while the above `flake` will still look for styling but is more permissive, this is recommended while transitioning to Py3k